lib/resolve.js

```diff
@@ -2,15 +2,44 @@
 
 const memoize = require('lodash.memoize')
 
+const addActionsHeader = function (headers) {
+  return Object.assign({'Accept': 'application/vnd.heroku+json; version=3.actions'}, headers || {})
+}
+
+const appAddon = function (heroku, app, id, options = {}) {
+  const headers = addActionsHeader(options.headers)
+  return heroku.post('/actions/addons/resolve', {
+    'headers': headers,
+    'body': {'app': app, 'addon': id, 'addon_service': options.addon_service || null}
+  })
+  .then((addons) => singularize(addons))
+}
+
+exports.appAddon = appAddon
+
 const addonResolver = function (heroku, app, id, options = {}) {
-  const headers = options.headers || {}
+  const headers = addActionsHeader(options.headers)
+
   let getAddon = function (id) {
-    return heroku.get(`/addons/${encodeURIComponent(id)}`, {headers})
+    return heroku.post('/actions/addons/resolve', {
+      'headers': headers,
+      'body': {'app': null, 'addon': id, 'addon_service': options.addon_service || null}
```

Comment:
@dickeyxxx I was unsure what `undefined` serialized to (which is not a great reason), so I tested it and it omits the key completely when serializing.  Removed the fallback in https://github.com/heroku/heroku-cli-addons/pull/57/commits/6c62a10faaf960778ebdc1213926d7f0fb989190